R/fotmob_helpers.R

```diff
@@ -8,14 +8,14 @@
   page %>%
     rvest::html_elements("script") %>%
     rvest::html_text(trim = TRUE) %>%
-    purrr::keep(stringr::str_detect, "lsLeague")
+    purrr::keep(stringr::str_detect, "props")
 }
 
 #' @importFrom stringr str_extract
 .fotmob_get_build_id <- function() {
   meta <- .fotmob_extract_meta()
   meta %>%
     stringr::str_extract('(?<=\\"buildId\\"[:]).*(?=\\,\\"isFallback\\")') %>%
-    .fromJSON()
-
+    safely_from_json() %>%
```

Comment:
using `safely_from_json()` here even though `.fromJSON()` was working fine here... seems like a good idea to use `safely_from_json()` in case `.fromJSON()` breaks but `jsonlite::fromJSON()` works